dbms/src/main/java/nl/nn/adapterframework/dbms/IDbmsSupport.java

```diff
@@ -43,14 +41,18 @@ public interface IDbmsSupport {
 	 * Numeric value defining database type, defined in {@link DbmsSupportFactory}.
```

Comment:
This JavaDoc comment is wrong. I know it's none of your fault but since we're doing some refactoring, do you mind please fixing it? (This is now an enum and no longer a numeric value, also it is not defined in that class it links to).